SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -308,6 +308,8 @@ FNetworkGUID FSpatialNetGUIDCache::RegisterNetGUIDFromPath(const FString& PathNa
 	FNetGuidCacheObject CacheObject;
 	CacheObject.PathName = FName(*PathName);
 	CacheObject.OuterGUID = OuterGUID;
+	CacheObject.bNoLoad = true;
+	CacheObject.bIgnoreWhenMissing = true;
```

Comment:
This looks good, but when I investigated this it looked like when issuing these cached objects on client, it doesn't always set them to true. Would be worth investigating the difference to ensure we don't hide any additional issues.

Reply:
+1